cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -188,14 +214,11 @@ <h2 class="h3 u-mb15">
                     Median purchase APR
                 {% endif %}
             </div>
-        {% else %}
-            <p>This card does not charge interest on purchases.</p>
-        {% endif %}
-        {% if card.purchase_apr_vary_by_credit_tier %}
             <p>
-                Your APR will be based on your credit score. You’ll only know your
-                exact APR if you are approved for the card. Here are average APRs
-                for different credit scores for this card:
+                {{'Your APR will be based on your credit score. ' if card.purchase_apr_vary_by_credit_tier }}
+                You’ll only know your exact APR if you are approved for the
+                card. Here are average APRs for different credit scores for this
```

Comment:
Per recent discussion, should we remove the word "average" here? This value is either:

- tier-specific median
- global median
- global maximum

so never actually an average. How about "typical APRs"?

Reply:
@chosak, fixed in b91b1f8